packages/nextjs/src/index.ts

```diff
@@ -0,0 +1,146 @@
+/**
+ * Fedify with Next.js
+ * ================
```

Comment:
It would look better: 🤣

```suggestion
 * Fedify with Next.js
 * ===================
```